### DISCUSSION TITLE
Sehe die Diagramme nicht

### DISCUSSION THREAD

**Discussion by bluespiano** [RaBa64/OpenDTU#4] [Q&A]
Hallo, danke erst einmal für die Umsetzung dieser Version.
Nach dem OTA Update mit dieser Version kann ich die Diagramme leider nicht sehen.
Dauert es etwas, bis es angezeigt wird oder muss ich ggf noch weitere Schritte tun?

**Comment by RaBa64** [comment]
Sind die Diagramme inzwischen sichtbar?
Falls nicht, dann bitte überprüfen, ob die Firewall ggf. die Verbindung zum Google Server blockiert.

**Reply by andy5macht** [reply]
Habe das selbe Problem. Habe per OTA vom "original" OpenDTU per update gewechselt. Sieht aus wie das Original - keine Diagramme sichtbar.
Ich sehe in der Firewall nicht, dass anfragen an/von Google kommen bzw. geblock werden. 
Wenn ich "Read Database" Klicke. Komm ich auf eine leere Seite...
![grafik](https://github.com/RaBa64/OpenDTU-Database/assets/33346822/243496a7-c0f3-48de-a8b5-75630ae7a60b)

**Reply by andy5macht** [reply]
Halt! Es funktioniert. Ich war nur zu ungeduldig. Es wäre sinnvoll erstmal ein paar Stunden zu warten, bis die ersten Daten abglegt werden...

**Reply by Kitegerd** [reply]
Ich sehe auch keine Diagramme.  Read Database ist auch nicht zu sehen. Update auch über OTA.
Schade

**Reply by RaBa64** [reply]
Wenn Du kein "Read Database" siehst, dann ist auch nicht die richtige Software drauf.
Versuch noch einmal die Installation über OTA oder falls es wiederholt nicht klappt, direkt über USB.

**Comment by Kitegerd** [comment]
Erase  / dann opendtu-generic.factory.bin  per usb geflasht.   Keine Änderung.   Chip - Modell ist ESP32-D0WD-V3

**Reply by EBergr** [reply]
ich habe auch das Modell und spiele die version 23.7.12 als original ein, ich erhalte Werte vom WR. spiele ich danach den Fork ein, habe ich keine DB und er findet keine Funkmodule. Mit Version 23.6.8 kann ich auch den Fork installieren, Funkmodule gehen auch aber er zeigt mir keine DB an, hat er also nicht installiert. Verstehe das Problem nicht

**Reply by RaBa64** [reply]
Zwischen diesen beiden Versionen wurden die Treiber für die NRF24L01+ Funkmodule geändert. Möglicherweise liegt es daran.
Ich selbst verwende ein CMT2300A, und habe deshalb das Problem mit dem NRF24L01+ nicht bemerkt.
Ich schaue mir das mit dem neuen NRF24L01+ Treiber genauer an, bin da aber auf eure Hilfe beim Testen angewiesen.
Danke vorab.

Was die Datenbank angeht, dann erfolgt ein Eintrag immer erst zur nächsten vollen Stunden.
Also erst wenn nach 1-2 Stunden immer noch kein Eintrag in der Datenbank ist, sollte das zu denken geben.
Der Link "Read Database" auf der Seite Info/System solle aber immer da sein.

**Reply by EBergr** [reply]
Mit der zweit ältesten Version geht es jetzt. Scheint an den Sendern zu liegen.

**Comment by Kitegerd** [comment]
0509952 funktioniert mit nRF24    Danke

**Reply by RaBa64** [reply]
Ich habe die Software auf den gleichen Stand gebracht wie das Original.

**Reply by andy5macht** [reply]
Danke 1a
